courses/Module 1/1-2-Reading Files.Rmd

```diff
@@ -50,7 +50,7 @@ crypto_art <- read.csv("crypto_art_pulse.csv")
 --
 
 ## Data in Excel
-Formats xls and csv - what's the difference? File extensions .xls or .xlsx are proprietary Excel formats, they are binary files .csv is the extension for comma separated value files. They are text files - and directly readable.
+Excel has file formats such as xls and csv - so what are the differences? File extensions .xls or .xlsx are proprietary Excel formats, they are binary files. .csv is the extension for comma separated value files. They are text files - and are directly readable.

```

Comment:
I love the little touches like this you added that make it so much more readable!